src/js_native_api_v8.cc

```diff
@@ -38,6 +38,17 @@
 #define CHECK_NEW_FROM_UTF8(env, result, str)                                  \
   CHECK_NEW_FROM_UTF8_LEN((env), (result), (str), NAPI_AUTO_LENGTH)
 
+#define CHECK_NEW_STRING_ARGS(env, str, length, result)                        \
```

Comment:
The change is already too big. I wonder if we can split up small improvements like this into a separate PR?
The same is about some changes in the docs.

Reply:
https://github.com/nodejs/node/pull/50664